lib/utils/normalizeOptions.js

```diff
@@ -14,7 +15,7 @@ function normalizeOptions(compiler, options) {
     : {};
 
   const defaultOptionsForStatic = {
-    directory: process.cwd(),
+    directory: path.resolve(process.cwd(), 'static'),
```

Comment:
I don't use check directory is exist to allow ability adding `static` dynamically and working with it without problems